aws/testdata/service/eks/node-group-launch-template-user-data.sh.tmpl

```diff
@@ -0,0 +1,8 @@
+MIME-Version: 1.0
```

Comment:
See above `ExpectError` comment. Theoretically this should be just a shell script, but the API is not validating things correctly on update. If MIME is not required on update, can remove the MIME wrapping here.

Reply:
Now that the documentation is live, we can see that the MIME multi-part bits are required so the user data can be merged. 👍